docs/DeveloperGuide.md

```diff
@@ -55,9 +55,40 @@ The `Ui` component
 1. The command execution can affect the `Model` (e.g. setting a value).
 1. In addition, the `Ui` may also perform certain actions, such as displaying help to the user.
 
-Given below is the Sequence Diagram for interactions within the `Logic` component for the `parse("set r 500")` API call.
+The various command classes to be parsed can be divided into Trivial and Non-Trivial classes:
```

Comment:
This would probably be better moved to its own section rather than being in the Architecture section.

Reply:
Remember to update the ToC once you do so 💯